lib/astutils.h

```diff
@@ -253,11 +253,7 @@ struct ReferenceToken {
     ErrorPath errors;
 };
 
-SmallVector<ReferenceToken> followAllReferences(const Token* tok,
-                                                bool temporary = true,
-                                                bool inconclusive = true,
-                                                ErrorPath errors = ErrorPath{},
-                                                int depth = 20);
+SmallVector<ReferenceToken> followAllReferences(const Token* tok, bool temporary = true);
```

Comment:
Since we are removing the `inconclusive` flag, then we should add that to the `ReferenceToken` so we can know if it was inconclusive or not.